Source/Scene/ModelExperimental/ModelExperimentalUtility.js

```diff
@@ -334,7 +335,9 @@ ModelExperimentalUtility.getCullFace = function (modelMatrix, primitiveType) {
  */
 ModelExperimentalUtility.sanitizeGlslIdentifier = function (identifier) {
   // Remove non-alphanumeric characters.
-  let sanitizedIdentifier = identifier.replaceAll(/[^A-Za-z0-9_]/g, "");
+  let sanitizedIdentifier = identifier.replaceAll(/[^A-Za-z0-9_]/g, "_");
+  // Replace multiple underscores.
+  sanitizedIdentifier = sanitizedIdentifier.replaceAll(/__+/g, "_");
```

Comment:
As we discussed offline this could be simplified to a single regex:

```js
let sanitizedIdentifier = identifier.replaceAll(/[^A-Za-z0-9]+/g, "_");
```